.travis.yml

```diff
@@ -63,7 +63,7 @@ matrix:
       addons:
           apt:
               sources: *all_sources
-              packages: ['clang-6.0', 'libstdc++-8-dev']
+              packages: ['clang-6.0', 'libstdc++-7-dev']
```

Comment:
Why did we move to `libstdc++-8-dev `?

Reply:
Because of the clang-6/libstdc++ issues with new libstdc++ (ambiguous `get`). The bug is fixed for clang-7 but I don't believe they intend to back port it.